### PR TITLE
Change the launchers to use the symlink

### LINKED ISSUE #448: How to get Unity Launcher for Visual Studio Code and Atom to respect .bashrc
I am using Ubuntu 16.04 in virtualbox. There is a bug in electron that causes visual studio code and atom to open with a blank screen with the gpu enabled. To stop this, instead of launching

`visual-studio-code` and `atom`

I need to launch the application as `visual-studio-code --disable-gpu` and `atom --disable-gpu`.

When I add aliases to the .bashrc file in my home directory,
~~~~~~~~~~
# Aliases for Visual Studio Code and Atom to disable gpu upon launch
alias visual-studio-code='visual-studio-code --disable-gpu'
alias atom='atom --disable-gpu'
~~~~~~~~~~

Visual Studio Code and Atom launch fine from the terminal. However, When I click the app icon, it still launches without disabling the gpu for the application. How do I address this?

### PR DESCRIPTION
**Pull request by LyzardKing**
Change the launchers to use the symlink instead of the full path
Add the full path as TryExec (mainly for the tests)
Fixes #448

### CODE REVIEW

**Comment by LyzardKing** [comment]
The second commit (Firefox) would solve #323

**Comment by LyzardKing** [comment]
@didrocks What do you think of this proposal?
It could fix a couple of issues

File: tests/medium/test_web.py [hunk 1]
```diff
@@ -65,17 +65,3 @@ def test_install_with_changed_download_page(self):
         umake_command = self.command('{} web phantomjs'.format(UMAKE))
         self.bad_download_page_test(umake_command, download_page_file_path)
         self.assertFalse(self.path_exists(self.exec_path))
-
-
-class VisualStudioCodeInContainer(ContainerTests, test_web.VisualStudioCodeTest):
```
Comment (didrocks) [inline tests/medium/test_web.py]: That should probably be in another PR? (the finale migation of vsc in IDE category)

File: umake/tools.py [hunk 2]
```diff
@@ -447,13 +448,16 @@ def add_env_to_user(framework_tag, env_dict):
...
+        profile_filepath = profile_filepath.replace("profile", "bashrc")
+
     with open(profile_filepath, "a", encoding='utf-8') as f:
         f.write(profile_tag.format(framework_tag))
         for env in envs_to_insert:
             value = envs_to_insert[env]
             logger.debug("Adding {} to user's {} for {}".format(value, env, framework_tag))
             export = ""
-            if env != "PATH":
+            if env != "PATH" and not env.startswith("alias "):
```
Comment (didrocks) [inline umake/tools.py]: Should we have a test for this case?

File: tests/medium/test_web.py (hunk 1 above)
Comment (LyzardKing) [reply tests/medium/test_web.py]: Yes..this was part of #442
It's fixed in my local branch...but it's not uploaded yet

File: umake/tools.py (hunk 2 above)
Comment (LyzardKing) [reply umake/tools.py]: We have now a simple test in test_tools.py

**Comment by LyzardKing** [comment]
Moved the alias setting in tools to the new firefox PR.
